### ISSUE TITLE
Bug: Highlighting does not work correctly when multiple verses are in check

### ISSUE THREAD

**Issue by PhotoNomad0** [unfoldingWord/translationCore#7337]
### User Story

Highlighting does not work correctly when multiple verses are in check

In php project in tN for ml/test_org2.  Words in the original quote for vs 1 are also highlighted in vs 5:

![Screen Shot 2022-06-30 at 8.43.07 AM.png](https://images.zenhubusercontent.com/564e463c638e718c995eb353/2621367c-a63d-4e7c-871f-0518c54dddab)

As a [type of user], I want [some goal] so that [some reason].

### Features / Specifications

- [ ]
- [ ]
- [ ]

### Definition of Done

- [ ]
- [ ]
- [ ]

**Comment by PhotoNomad0** [comment]
To test, in content updates check option `show pre-release resources`; download the ult, ta and tn resources from ml/test_org2.  Open a php project and for tN select ml/test_org2, scroll down to Assumed knowledge, ref 2:1-2,5.  make sure the highlighting is only in the first verse.

**Comment by PhotoNomad0** [comment]
Note: Still seeing problems in latest build.  GL quote has extra ellipsis and ULT has extra `in Christ` highlighted in verse 5.

check is in `~/translationCore/resources/ml/translationHelps/translationNotes/v0.5.1_test_org2/culture/groups/php/figs-explicit.json` and looks fine:

```
  {
    "comments": false,
    "reminders": false,
    "selections": false,
    "verseEdits": false,
    "nothingToSelect": false,
    "contextId": {
      "checkId": "b1q7",
      "occurrenceNote": "“If Christ has encouraged you” or “If you are encouraged because of Christ”",
      "reference": {
        "bookId": "php",
        "chapter": 2,
        "verse": "1-2,5"
      },
      "tool": "translationNotes",
      "groupId": "figs-explicit",
      "quote": [
        {
          "word": "εἴ",
          "occurrence": 1
        },
        {
          "word": "τις",
          "occurrence": 1
        },
        {
          "word": "…"
        },
        {
          "word": "παράκλησις",
          "occurrence": 1
        },
        {
          "word": "ἐν",
          "occurrence": 1
        },
        {
          "word": "Χριστῷ",
          "occurrence": 1
        }
      ],
      "quoteString": "εἴ τις … παράκλησις ἐν Χριστῷ",
      "glQuote": "",
      "occurrence": 1
    }
  },
```

<img width="826" alt="Screen Shot 2022-08-03 at 7 38 30 AM" src="https://user-images.githubusercontent.com/14238574/182600315-de8fd706-aef6-4ba3-b27f-77efb6dc2b20.png">

**Comment by PhotoNomad0** [comment]
@elsylambert Fixed in develop build translationCore 3.3.1 (eb96f38) 

To test, in content updates select option `show pre-release resources`; download the ult, ta and tn resources from ml/test_org2. Open a php project and for tN select ml/test_org2, scroll down to `Assumed knowledge...`, ref 2:1-2,5. make sure the highlighting is only in the first verse.  Make sure action area shows correct GL quote:

![Screen Shot 2022-08-03 at 2 57 43 PM](https://user-images.githubusercontent.com/14238574/182689398-c36f70c2-0625-452e-be10-03f00fe3beca.png)

**Comment by elsylambert** [comment]
Highlight works as expected in translationCore 3.4.0 (551be4e).
 
<img width="1492" alt="Screen Shot 2022-09-22 at 9 15 07 AM" src="https://user-images.githubusercontent.com/51806180/191756913-097e913c-5b4c-49db-8bbc-64d167fc4d36.png">
